test.sh

```diff
@@ -19,6 +19,28 @@ for dir in ${VERSIONS}; do
     VERSION=$dir test/run
   fi
 
+  if [ -n "${TEST_CONU_MODE}" ]; then
+    if [[ -x test/run-conu ]]; then
+      docker run \
+        --net=host \
+        -e VERSION=$dir \
+        -e IMAGE_NAME \
+        --rm -v /dev:/dev:ro \
+        -v /var/lib/docker:/var/lib/docker:ro \
+        --security-opt label=disable \
+        --cap-add SYS_ADMIN \
+        -ti \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v ${PWD}/../:/src \
+        -w /src/${dir}/ \
+        -ti \
+        docker.io/usercont/conu:0.6.2 \
```

Comment:
Don't you want to have some floating docker tag here instead of 0.6.2?

Reply:
Actually not: in case we accidentally create a breaking change in a new version, we don't want all your conu tests to start failing. I think it's a better approach to systematically update to newer versions once you're sure they work as you expect. Using latest greatest might be pretty adventurous, especially when the container ecosystem is still changing rapidly.

But your call, really.

Reply:
> Using latest greatest might be pretty adventurous

By floating tag I did not mean "latest greatest", but rather "latest stable".  Whether we'll update the version here all the time by PR, or you'll tag it into "conu:stable" tag doesn't make much practical difference - it will hurt us and  PR is only additional work.  Btw.,  conu should simplify our lives so we don't have to take care of ever-changing container API, not that we'll have to take care ever-changing API in conu.